packages/dendron-next-server/components/graph.tsx

```diff
@@ -1,12 +1,7 @@
 import { createLogger, engineSlice } from "@dendronhq/common-frontend";
 import _ from "lodash";
 import React, { useEffect, useRef, useState } from "react";
-import cytoscape, {
-  Core,
-  EdgeDefinition,
-  ElementsDefinition,
-  EventHandler,
-} from "cytoscape";
+import cytoscape, { Core, EdgeDefinition, EventHandler } from "cytoscape";
 // @ts-ignore
 import euler from "cytoscape-euler";
```

Comment:
there's no `@types/` package for this?

Reply:
There's not. I can look into adding a `d.ts` file to be able to get rid of the `//@ts-ignore` if you'd like

Reply:
yeah, that would be preferred :)